Makefile

```diff
@@ -87,6 +87,7 @@ tidy:
 .PHONY: clean
 clean:
 	@$(shell find ./example -type f -name "controller-registration.yaml" -exec rm '{}' \;)
+	@$(shell find ./example -type f -name "extension.yaml" -exec rm '{}' \;)
```

Comment:
Don't we need to rm `example/extension/base/extension.yaml` as well? 😅 

At least, we did it for the registry-cache: ref https://github.com/gardener/gardener-extension-registry-cache/pull/448/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R66

Reply:
The command here already deletes both files. I missed to adjust the registry cache extension after I force pushed, so it's currently redundant there.

Reply:
I see, thanks for explaining! I will raise a PR for the registry-cache extension to unify. 👍

Reply:
Ref https://github.com/gardener/gardener-extension-registry-cache/pull/451